index.html

```diff
@@ -47,7 +113,6 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
```

Comment:
...
suggestion
   <h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
....